appdirs.py

```diff
@@ -144,7 +150,7 @@ def site_data_dir(appname=None, appauthor=None, version=None, multipath=False):
     else:
         # XDG default for $XDG_DATA_DIRS
         # only first, if multipath is False
-        path = os.getenv('XDG_DATA_DIRS',
+        path = getenv('XDG_DATA_DIRS',
                          os.pathsep.join(['/usr/local/share', '/usr/share']))
```

Comment:
This is mis-indented I believe.

Reply:
I tried to minimise changes, but yes, it's probably worth fixing the original indentation while I'm here. I would usually let black handle it :)